sqldelight-compiler/src/test/kotlin/com/squareup/sqldelight/core/tables/InterfaceGeneration.kt

```diff
@@ -158,7 +159,101 @@ class InterfaceGeneration {
       |        |  longValue: ${"$"}longValue
       |        |  floatValue: ${"$"}floatValue
       |        |  doubleValue: ${"$"}doubleValue
-      |        |  blobValue: ${"$"}blobValue
+      |        |  blobValue: ${"$"}{blobValue.kotlin.collections.contentToString()}
```

Comment:
This doesn't get imported since it's not inside a `FileSpec`, other tests put the generated `TypeSpec` into the `FileSpec` to get proper imports